conf.py

```diff
@@ -110,3 +110,16 @@
 # Exclude class reference when marked with tag i18n.
 if is_i18n:
     exclude_patterns = ['classes']
+    # Add remaps here if locale code differs on Weblate and RTD
+    # Used to generate language status badge
+    weblate_remaps = {
+        'zh_CN': "zh_Hans",
```

Comment:
For the reference, there are more remaps needed in theory, but I only add the one relevant for our current set of localized instances: https://github.com/godotengine/godot-docs-l10n/blob/master/build_langs.txt

Reply:
I've changed Weblate to accept original language codes here as well, so you can remove the remapping now :-). See https://github.com/WeblateOrg/weblate/commit/aa360585647593bcfffd5974c5766be3b63f3108

Reply:
Thanks!